docker/server/Dockerfile.ubuntu

```diff
@@ -3,8 +3,6 @@ FROM ubuntu:20.04
 # see https://github.com/moby/moby/issues/4032#issuecomment-192327844
 ARG DEBIAN_FRONTEND=noninteractive
 
-COPY su-exec.c /su-exec.c
-
 # ARG for quick switch to a given ubuntu mirror
 ARG apt_archive="http://archive.ubuntu.com"
```

Comment:
`archive.ubuntu.com` looks completely misleading, there is no chance find appropriate ClickHouse version there.